modules/navigation/3d/nav_mesh_generator_3d.cpp

```diff
@@ -933,9 +934,10 @@ void NavMeshGenerator3D::generator_bake_from_source_geometry_data(Ref<Navigation
 			nav_indices.write[1] = recast_index_to_native_index[index2];
 			nav_indices.write[2] = recast_index_to_native_index[index3];
 
-			p_navigation_mesh->add_polygon(nav_indices);
+            nav_polygons.push_back(nav_indices);
```

Comment:
```suggestion
			nav_polygons.push_back(nav_indices);
```
Wrong indentation type